Workade/Share/RegionButton.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  LocationButton.swift
+//  Workade
+//
+//  Created by Hong jeongmin on 2022/11/22.
+//
+
+import UIKit
+
+class RegionButton: UIButton {
```

Comment:
[P1] 더 이상 상속하지 않는다면 final 선언 부탁드립니다.
```suggestion
final class RegionButton: UIButton {
```

Reply:
수정했습니다!!